README.md

```diff
@@ -3,95 +3,20 @@ PRX Metrics Frontend App
 
 This app is used to view podcast metrics.
 
-# Install
+## Local Development
 
-## API and Backend Dependencies
+See [CONTRIBUTING](/PRX/metrics.prx.org/blob/master/CONTRIBUTING.md) for information on installing and configuring this project's dependencies, such as [ID](https://github.com/PRX/id.prx.org/) and [CMS](https://github.com/PRX/cms.prx.org/).
```

Comment:
Honestly, seems a bit weird to me to have a 20 line README, and a 55 line CONTRIBUTING.  When the info is that short, I'd prefer a single file with a `# Contributing` section.